channel/channel.go

```diff
@@ -0,0 +1,24 @@
+package channel
+
+import (
+	"math/big"
+
+	"github.com/statechannels/go-nitro/channel/state"
+)
+
+type Channel interface {
```

Comment:
I believe we may indeed want something like this interface -- but I think it is unreferenced at the moment, so should probably be cut?